### PR TITLE
chore(admin-ui): Add pt-BR to supported languages

### PR DESCRIPTION
**Pull request by olivermrbl**
Not sure the display name is correct, but should do the job

### CODE REVIEW

File: packages/admin-ui/ui/src/i18n/index.ts [hunk 1]
```diff
@@ -12,6 +12,10 @@ export const supportedLanguages = [
     locale: "fr",
     name: "Français",
   },
+  {
+    locale: "pt-BR",
+    name: "Português Brasileiro",
```
Comment (kasperkristensen) [inline packages/admin-ui/ui/src/i18n/index.ts]: Nit: Usually I think you differentiate different locales of a language as `Portuguese (Brazil)` and `Portuguese (Portugal)`, etc.

**Review by kasperkristensen — APPROVED** [review]
LGTM, with a tiny nit

File: packages/admin-ui/ui/src/i18n/index.ts (hunk 1 above)
Comment (olivermrbl) [reply packages/admin-ui/ui/src/i18n/index.ts]: We decided to display the name as used in the given language. We might add another prop `en_name`, that we can use in a tooltip or similar. 

That said, I am up for going with your suggestion, if you think it's better.

Comment (carlos-r-l-rodrigues) [reply packages/admin-ui/ui/src/i18n/index.ts]: Probably another property like `name_en` is a better choice. For the name I would change it to `Português (Brasil)`

Comment (olivermrbl) [reply packages/admin-ui/ui/src/i18n/index.ts]: Got it thanks, I'll change the name for now, and we can think about adding the English version in a follow up 👍

**Comment by Daniel-Boll** [comment]
Are you referring to the `display name` field in this JSON object?

```json
{
  "name": "Português (Brasil)"
}
```

If that's the case, both `Português (Brasil)` and `Português Brasileiro` are standard and interchangeable terms for us.

**Pull request merged**